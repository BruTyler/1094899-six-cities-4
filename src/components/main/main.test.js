import React from 'react';
import renderer from 'react-test-renderer';
import Main from './main.jsx';

const APARTMENTLIST = [`apart1`, `apart2`];
const CLICKHANDLER = () => {};

describe(`Main render suit`, () => {
  it(`Main render apartment list case`, () => {
    const generatedTree = renderer.create(
        <Main
          apartmentList={APARTMENTLIST}
          onApartmentTitleClick={CLICKHANDLER}
        />
    ).toJSON();

    expect(generatedTree).toMatchSnapshot();
  });
});
