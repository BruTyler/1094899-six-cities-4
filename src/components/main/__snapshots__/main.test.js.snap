// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main render suit Main render apartment list case 1`] = `
Array [
  <p>
    Предложения аренды:
  </p>,
  <div>
    <img
      height="40"
      src="img/no-places.png"
      width="50"
    />
    <a
      className="apartment__title"
      href="#"
      onClick={[Function]}
    >
      apart1
    </a>
  </div>,
  <div>
    <img
      height="40"
      src="img/no-places.png"
      width="50"
    />
    <a
      className="apartment__title"
      href="#"
      onClick={[Function]}
    >
      apart2
    </a>
  </div>,
  <hr />,
  <p>
    Всего предложений: 
    2
  </p>,
]
`;
